### DISCUSSION TITLE
Warning about MQTT on Public Channels

### DISCUSSION THREAD

**Discussion by realitytech** [meshtastic/firmware#3572] [Ideas]
As per my other post we have an issue with MQTT spamming LongFast on our mesh.
All the tutorials I've seen are clear this is a REALLY bad idea however it seems people arent listening.
Could there be a confirmation prompt/warning in the various UIs about not enabling the downlink on public channels?

**Comment by GUVWAF** [comment]
Make sure you have "ignore MQTT" enabled in your LoRa settings. If your nodes and the other nodes are using recent firmware (2.2.19 or higher), it will simply ignore all packet coming from MQTT.

Further limiting MQTT downlink is discussed in https://github.com/meshtastic/rfcs/pull/9.

**Reply by realitytech** [reply]
I think these two posts kind happened at the time. We've implemented this in our mesh along with our routers carrying a "please dont..." in their IDS and people rolling up firmwares and new people using the new firmware seems to have solved it. The issue we started to see is that there are "islands" where people did enable it and no-one had ignore MQTT on. Its a pain but yes manageable now after the lates firmwares are in play. Thank you
